content/project-news/hermes-belusca-hired-full-time.md

```diff
@@ -0,0 +1,27 @@
+---
+title:       "Hermès BÉLUSCA - MAÏTO hired full-time to work on the ReactOS GUI Setup"
+author:      "GeoB99"
+date:        2023-09-28
+---
+
+Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès BÉLUSCA - MAÏTO to work full-time on the ReactOS GUI Setup for the next 3 months.
+
+Hermès is a long-time contributor and developer who worked on the project since 2012.
+He has several skills and expertise into various components of ReactOS, notably being the Client/Server Runtime Subsystem (CSRSS), Console Server Subsystem, NTVDM, and others.
+
+His upcoming goal is to finish the first-stage ReactOS GUI Setup, that offers an alternative to the classic first-stage text-mode setup (or also called USETUP).
+Currently the only option to install ReactOS is through the text-mode setup, with the rest of the setup installation being handled after the second-stage setup.
+This has been and still is the standard method of installing ReactOS on a VM or hardware since the inception of the project.
```

Comment:
This currently is the primary method of installing ReactOS On a VM

Reply:
I wouldn't strip the "on hardware" here. It's not that a severe overstatement as we used to dare in earlier announcements. ;-)

Reply:
The biggest point was “this has been and still is the standard…” sounds a little odd. But yes for sure don’t strip out the hardware mention